src/sage/modular/abvar/abvar.py

```diff
@@ -91,6 +91,9 @@ def is_ModularAbelianVariety(x) -> bool:
 
         sage: from sage.modular.abvar.abvar import is_ModularAbelianVariety
         sage: is_ModularAbelianVariety(5)
+        doctest:warning...
+        DeprecationWarning: the function is_ModularAbelianVariety is deprecated; use 'isinstance(..., ModularAbelianVariety_abstract)' instead
+        See https://github.com/sagemath/sage/issues/38035 for details.
         False
```

Comment:
The same comment on capitalization and period?

Reply:
Thanks. 

This may be common pitfall since the style is different from that of error messages. It seems that deprecation warning needs to be accompanied with an explanation, which is often  a few sentences long, and hence the style.